lib/stimulus_reflex/broadcasters/broadcaster.rb

```diff
@@ -9,7 +9,7 @@ class Broadcaster
 
     def initialize(reflex)
       @reflex = reflex
-      @logger = Rails.logger
+      @logger = Rails.logger if defined?(Rails.logger)
```

Comment:
Would safe navigation work here instead of a defined check?

Reply:
Possibly, I will try it out!

Reply:
Alas, doesn't work :-(

Reply:
The problem is, I can't find a way to stub **module** methods with minitest... gives me `NameError: undefined method `logger' for class `Rails'` when I try ... well, because it's no class method, but a module method...